llvm/lib/Target/RISCV/RISCVSchedSiFive7.td

```diff
@@ -966,14 +966,14 @@ def : InstRW<[WriteIALU], (instrs COPY)>;
 // VCIX
 //
 // In principle we don't know the latency of any VCIX instructions. But instead
-// of taking the default of 1, which can lead to issues [1], we assume that they
-// have a fairly high latency.
+// of taking the default of 1, which can lead to issues [1], we use the default
+// latency provided by `SiFive7GetCyclesDefault`.
```

Comment:
nit: it would be nice to clarify that using SiFive7GetCyclesDefault can still lead to issues, but those issues are less severe than the issues that come with using a default of 1.

Reply:
Isn't the behavior of SiFive7GetCyclesDefault LMul*2 not 1.?

Reply:
I was referring to the "default" of 1 from earlier in the comment, not to be confused with `SiFive7GetCyclesDefault`:

> instead of taking the default of 1

Reply:
I expanded the comment a bit. Let me know if that's what you had in mind! 🙂